src/sitemap-pipe.js

```diff
@@ -34,8 +76,6 @@ import { PipelineResponse } from './PipelineResponse.js';
 export async function sitemapPipe(state, req) {
   const { log } = state;
   state.type = 'sitemap';
-  // force loading from preview
-  state.partition = 'preview';
```

Comment:
removed that again, or we lose the information on the `origin` to use in generated `sitemap.xml`

Reply:
but it important that the sitemap.xml and/or sitemap.json is read from preview